readthedocs/search/faceted_search.py

```diff
@@ -153,8 +159,8 @@ class PageSearchBase(RTDFacetedSearch):
     doc_types = [PageDocument]
     index = PageDocument._doc_type.index
 
-    _outer_fields = ['title^4']
-    _section_fields = ['sections.title^3', 'sections.content']
+    _outer_fields = ['title^2']
+    _section_fields = ['sections.title^4', 'sections.content']
```

Comment:
didn't see much different between the new and old values, but it does make sense to have more priority on the section title.

Reply:
I think 3 is probably fine, still. I think we want to be careful with boosting -- I wonder if we could just get rid of them in general? My understanding is that ES already weights shorted fields a bit higher, and it will let the internal functions of ES work a bit better?

Reply:
Reverted to 3, I'll take a look how this looks like without boosting